Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-grid/SubmissionsGrid.module.scss

```diff
@@ -1,6 +1,7 @@
 @use 'src/styles/colors';
+@use 'src/styles/fonts';
 
-.submissionRow {
+.submissionRow {  
```

Comment:
Invalid? There should be a stylelint rule for that, i.e. unused `use`. Please check and if the rule is missing, log it

Reply:
https://github.com/SoftUni-Internal/exam-systems-issues/issues/518